src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,93 @@
+package duke.storage;
+
+import duke.task.TaskList;
+import duke.exception.DukeException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Storage {
+    private static final String DIR = System.getProperty("user.dir");
```

Comment:
Since the scope of this variable is relatively wide, should this variable have a longer and more descriptive name like `DIRECTORY`?